src/ore/Cargo.toml

```diff
@@ -37,6 +38,7 @@ pin-project = "1"
 prometheus = { git = "https://github.com/MaterializeInc/rust-prometheus.git", default-features = false, optional = true }
 smallvec = { version = "1.7.0", optional = true }
 stacker = { version = "0.1.14", optional = true }
+timely = { git = "https://github.com/TimelyDataflow/timely-dataflow", default-features = false, features = ["bincode"] }
```

Comment:
I'd like to keep timely stuff out of `ore`. The rule of thumb with `ore` is that only things that would go in the standard library in Go or Python belong in `ore`. (The only thing that's really in violation of this right now is `prometheus`; I've been meaning to split out a `metrics` crate for a while.)

Can you create a new `timely-util` crate a la `aws-util` or `kafka-util` instead?

Reply:
Oops! That was my suggestion. `timely-util` sounds fine to me